src/components/FlatmapVuer.vue

```diff
@@ -394,9 +394,9 @@
               </el-option>
             </el-select>
           </el-row>
-          <el-row class="backgroundSpacer"></el-row>
-          <el-row class="backgroundText" v-if="isFC">Dimension display</el-row>
-          <el-row class="backgroundControl" v-if="isFC">
+          <el-row class="backgroundSpacer" v-if="displayFlightPathOption"></el-row>
+          <el-row class="backgroundText" v-if="displayFlightPathOption">Dimension display</el-row>
```

Comment:
Perhaps we should renamed Dimension display to Flight path display?

Reply:
The names are updated.